ui/browser/views/tabbar/tabbar.jsx

```diff
@@ -13,7 +13,7 @@ specific language governing permissions and limitations under the License.
 import React, { PropTypes } from 'react';
 import Tab from './tab.jsx';
 import { menuTabContext } from '../../actions/external';
-import { createTab, closeTab, setCurrentTab } from '../../actions/main-actions';
+import { createTab, closeTab, clickTab } from '../../actions/main-actions';
```

Comment:
using click instead of an action would be most questionable change here - deviating from the very explicit actions being performed.